scrapers/GammaEntertainment.yml

```diff
@@ -138,14 +138,7 @@ xPathScrapers:
       Movies:
         Name: $info/a/img/@title|//a[@class="sceneLink  "]/@title|//a[@class="dvdLink  "]/@title
       Image:
-        selector: //script[contains(text(), 'picPreview')]/text()
-        replace:
-          - regex: .+(?:picPreview":")([\w:]+)(?:[\\\/]+)([\w-\.]+)(?:[\\\/]+)(\w+)(?:[\\\/]+)(\d+)(?:[\\\/]+)([\d_]+)(?:[\\\/]+)(\w+)(?:[\\\/]+)(\d+)(?:[\\\/]+)(\d+)(?:[\\\/]+)([\w]+)(?:[\\\/]+)([\w.]+).+
-            with: $1//$2/$3/$4/$5/$6/$7/$8/$9/$10
-            # if using the transport subdomain, parameters need to be passed
-            # otherwise a cropped square image is returned by default
-          - regex: (https:\/\/transform.+)
-            with: $1?width=960&height=543&enlarge=true
+        selector: //meta[@name="twitter:image"]/@content
```

Comment:
i haven't tested in all sites 
but at least in `evilangel` the `meta` for the image is missing
sample urls
```
https://www.evilangel.com/en/video/Anal-Fuck-Cunt-Creampie-For-Karma-Rx/168306
https://www.evilangel.com/en/video/I-Am-Angela-Ep-1-The-Journey/134756
https://www.evilangel.com/en/video/I-Am-Angela-Ep-4-Ciao-Bella/134759
https://www.evilangel.com/en/video/Lilly-Loves-Anal-Fun--DickDildo-DP/132082
https://www.evilangel.com/en/video/Panty-Pops/27598
```
that image is missing using the `new` selector

Reply:
Evilangel/Wicked will be on the CDP version of GammaEntertainment with #127 so it's fix this issue.
Will check every site listed to see which work.

Reply:
So after checking, like half work with the meta selector.
I revert this change because he will take a long time trying to get a scraper working for all this.